datasets/modeled/transport/model_year_to_model_year.csv

```diff
@@ -0,0 +1,11 @@
+from_id,to_id,from_fraction
+2025,2024,1
+2025,2026,0.8
+2030,2026,0.2
+2025,2028,0.4
+2030,2028,0.6
```

Comment:
Similarly, I wonder if we should be doing a non-linear interpolation since EVs are on an exponential adoption trajectory? Maybe the Task 2 data can be leveraged here, assuming that it's available for more model years than every 5 years?

Reply:
now interpolated/extrapolated based on DECARB 2 total